test/behaviour/concept/serialization/SerializationTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.vaticle.typedb.client.test.behaviour.concept.serialization;
+
+import com.vaticle.typedb.client.test.behaviour.BehaviourTest;
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        strict = true,
+        plugin = "pretty",
+        glue = "com.vaticle.typedb.client.test.behaviour",
+        features = "external/vaticle_typedb_behaviour/concept/serialization.feature",
+        tags = "not @ignore and not @ignore-typedb"
```

Comment:
Can you double check the `@ignore-typedb` is correct here? I think we have one for client-java or clients specifically or something? Might as well do a drive-by of the other BDD tests in this repo while you're at it